scripts/verify_build

```diff
@@ -14,8 +14,9 @@ export HAB_NOCOLORING=true
 export HAB_LICENSE="accept-no-persist"
 
 sudo -E hab pkg install core/ruby
-export PATH=$PATH:$(hab pkg path core/ruby)/bin
-sudo -E $(hab pkg path core/ruby)/bin/gem install toml
+export PATH
+PATH="$PATH:$(hab pkg path core/ruby)/bin"
```

Comment:
Do you know why it recommends splitting the export and the assignment? I'm curious and I don't see any explanations of things on the shellcheck homepage.

Reply:
I do!  It has to do with an interaction with `set -e` the "exit on error" option.  Consider:

```
> cat ./test.sh
#!/bin/bash

set -e

export PATH=$(false)
echo "got here"
> ./test.sh
got here
> cat test2.sh 
#!/bin/bash

set -e

export PATH
PATH=$(false)
echo "got here"
> ./test2.sh
>
```

When you do `export FOO=$(some_command)` or `local FOO=$(some_command)` you end up not seeing the failure some_command.

Reply:
The shellcheck wiki entry for it is here: https://github.com/koalaman/shellcheck/wiki/SC2155

Reply:
Oh neat, thanks!